include/bluetooth/mesh/scene_cli.h

```diff
@@ -0,0 +1,294 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/**
+ * @file
+ * @defgroup bt_mesh_scene_cli Scene Client model
+ * @{
+ * @brief API for the Scene Client model.
+ */
+
+#ifndef BT_MESH_SCENE_CLI_H__
+#define BT_MESH_SCENE_CLI_H__
+
+#include <bluetooth/mesh/scene.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+struct bt_mesh_scene_cli;
+
+/** @def BT_MESH_MODEL_SCENE_CLI
+ *
+ *  @brief Scene Client model composition data entry.
+ *
+ *  @param[in] _cli Pointer to a @ref bt_mesh_scene_cli instance.
+ */
+#define BT_MESH_MODEL_SCENE_CLI(_cli)                                          \
+	BT_MESH_MODEL_CB(BT_MESH_MODEL_ID_SCENE_CLI, _bt_mesh_scene_cli_op,    \
+			 &(_cli)->pub,                                         \
+			 BT_MESH_MODEL_USER_DATA(struct bt_mesh_scene_cli,     \
+						 _cli),                        \
+			 &_bt_mesh_scene_cli_cb),
+
+/** Scene state */
+struct bt_mesh_scene_state {
+	/** Status of the previous operation. */
+	enum bt_mesh_scene_status status;
+	/** Current scene, or @ref BT_MESH_SCENE_NONE if no scene is active. */
+	uint16_t current;
+	/** Target scene, or @ref BT_MESH_SCENE_NONE if no transition is in
+	 *  progress.
+	 */
+	uint16_t target;
+	/** Remaining time of the scene transition in milliseconds. */
+	uint32_t remaining_time;
+};
+
+/** Scene register parameters */
+struct bt_mesh_scene_reg {
+	/** Status of the previous operation. */
+	enum bt_mesh_scene_status status;
+	/** Current scene. */
+	uint16_t current;
+	/** Number of entries in the scenes list. */
+	size_t count;
+	/** Optional list of scenes to populate. */
+	uint16_t *scenes;
+};
+
+/** Scene client model instance. */
+struct bt_mesh_scene_cli {
+	/** @brief Status message callback.
+	 *
+	 *  @param[in] cli   Scene client model.
+	 *  @param[in] ctx   Message context parameters.
+	 *  @param[in] state Received scene state.
+	 */
+	void (*status)(struct bt_mesh_scene_cli *cli,
+		       struct bt_mesh_msg_ctx *ctx,
+		       const struct bt_mesh_scene_state *state);
+
+	/** @brief Scene register message callback.
+	 *
+	 *  @param[in] cli Scene client model.
+	 *  @param[in] ctx Message context parameters.
+	 *  @param[in] reg Received scene register.
+	 */
+	void (*reg)(struct bt_mesh_scene_cli *cli, struct bt_mesh_msg_ctx *ctx,
```

Comment:
register would be better I think, we don't abbreviate register elsewhere.

Reply:
register is a reserved word in c though 🤔

Reply:
Are you gonne let that stop you? :P

Not a big deal, I just find reg a bit too short, and doesn't match up well with the rest, maybe registered?

Reply:
I changed this and the scene_register structure to spell out scene_register, since both `register` and `reg` is pretty overloaded.

Reply:
Good call.

I fixed the doc errors. :)